superset/models/__init__.py

```diff
@@ -14,12 +14,9 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from . import (
```

Comment:
it is ok to declare here the nested imports, this is the purpose of init file,
solving circular imports should not be here